helpers/middlewares.js

```diff
@@ -0,0 +1,8 @@
+exports.exceptionMiddleware = async function exceptionMiddleware(err, req, res, next) {
+    res.status(500);
+    res.send({
+        'contextId': err.contextId,
+        'pageId': err.pageId,
+    });
+    next(err.error);
```

Comment:
There may be errors not originating from the action function: missing context/page, invalid function for custom action, etc. Such errors are consumed and not logged.